### ISSUE TITLE
Not install ungoogled-chromium to linux mint 18.3

### ISSUE THREAD

**Issue by JanuszSpiewa** [Eloston/ungoogled-chromium#1129]
I have install ungoogled-chromium 

Instalation falied requied  dependences libvacodec57 >7:3.4.6 and libvacodec-extra 7:3.4.6

My distribution does not have such packages in apt and I have not found any download anywhere on the internet

**Comment by JanuszSpiewa** [comment]
don't install from apt 
`ungoogled-chromium: Depends: libavcodec58 (> = 7: 4.0) but cannot be installed
                      Depends: libavformat58 (> = 7: 4.1) but cannot be installed
                      Depends: libavutil56 (> = 7: 4.0) but is not installable
                      Requires: libc6 (> = 2.28) but 2.23-0ubuntu11.2 is to be installed
                      Depends: libevent-2.1-6 (> = 2.1.8-stable) but cannot be installed
                      Requires: libfontconfig1 (> = 2.12.6) but 2.11.94-0ubuntu1.1 is to be installed
                      Requires: libgtk-3-0 (> = 3.19.12) but 3.18.9-1ubuntu3.3 is to be installed
                      Requires: libharfbuzz0b (> = 2.2.0) but 1.0.1-1ubuntu0.1 is to be installed
                      Depends: libicu63 (> = 63.1-1 ~) but cannot be installed
                      Depends: libjpeg62-turbo (> = 1.5.0) but cannot be installed
                      Requires: libjsoncpp1 (> = 1.7.4) but 1.7.2-1 is to be installed
                      Depends: libopenjp2-7 (> = 2.2.0) but it will not be installed
                      Depends: libre2-5 (> = 20160901) but cannot be installed
                      Requires: libstdc ++ 6 (> = 7) but 5.4.0-6ubuntu1 ~ 16.04.12 is to be installed
                      Depends: libvpx5 (> = 1.6.0) but cannot be installed
                      Depends: libwebp6 (> = 0.5.1) but cannot be installed
                      Depends: libwebpdemux2 (> = 0.5.1) but cannot be installed
                      Depends: libwebpmux3 (> = 0.6.1-2) but cannot be installed
E: Could not fix problems, damaged packages stopped.`
`

**Comment by Eloston** [comment]
Which package are you using? We currently support Debian and Ubuntu. You could try to pick a Debian or Ubuntu version that's closest to your Linux Mint version and see if the dependencies are compatible.

**Comment by JanuszSpiewa** [comment]
im using linux mint 18.3 based to ubuntu 16

**Comment by JanuszSpiewa** [comment]
I tried the repository for debian and ubuntu and on my distribution I cannot fulfill the dependencies

**Comment by r4sas** [comment]
Please give `cat /etc/os-release` output.

**Comment by Eloston** [comment]
Also, it seems you misunderstood my question. Let me rephrase: For the package you chose to install, which distribution (Debian or Ubuntu) and which version of that distribution (e.g. buster, focal, etc.) did you try to install?
